version.c

```diff
@@ -101,36 +110,16 @@ Init_version(void)
 void
 Init_ruby_description(void)
 {
-    VALUE description;
-
-    if (MJIT_OPTS_ON) {
-        description = MKSTR(description_with_mjit);
-    }
-    else if (rb_yjit_enabled_p()) {
-        description = MKSTR(description_with_yjit);
-    }
-    else {
-        description = MKSTR(description);
-    }
-
     /*
      * The full ruby version string, like <tt>ruby -v</tt> prints
      */
-    rb_define_global_const("RUBY_DESCRIPTION", /* MKSTR(description) */ description);
+    rb_define_global_const("RUBY_DESCRIPTION", rb_obj_freeze(rb_str_new2(rb_description())));
```

Comment:
Since `rb_description()` returns a C string, we can use `rb_str_new_static` instead.

```suggestion
    rb_define_global_const("RUBY_DESCRIPTION", rb_obj_freeze(rb_str_new_static(rb_description())));
```

Reply:
I remember now I wasn't using that because it requires the length to be calculated manually.